spec/requests/mutations/register_spec.rb

```diff
@@ -128,7 +128,7 @@
 
     it 'skips the register mutation' do
       expect(json_response[:errors]).to contain_exactly(
-        hash_including(message: "Field 'adminRegister' doesn't exist on type 'Mutation'")
+        hash_including(message: a_string_including("Field 'adminRegister' doesn't exist on type 'Mutation'"))
```

Comment:
graphql 2.4 adds additional information to the error message. But we only care in the spec about the field not being defined in the API